README.md

```diff
@@ -1,33 +1,63 @@
-# Using this template
+# seidemann-lab-to-nwb
+NWB conversion scripts for Seidemann lab data to the [neuro data without borders](https://www.nwb.org/) data format.
```

Comment:
```suggestion
NWB conversion scripts for Seidemann lab data to the [Neurodata Without Borders](https://www.nwb.org/) data format.
```

Probably want to propagate this to the cookiecutter, too.

Reply:
Also, maybe lead to the NWB Overview docs instead of the website (which isn't quite as up to date)

Reply:
https://nwb-overview.readthedocs.io/en/latest/intro_to_nwb/1_intro_to_nwb.html

Reply:
Done.

Reply:
"Neurodata Without Border"?

Reply:
Thanks, Ben just mentioned that.